.codecov.yml

```diff
@@ -244,38 +237,26 @@ component_management:
         # `auto` will use the coverage from the base commit (pull request base
         # or parent commit) coverage to compare against.
         target: auto
-        threshold: 2%
+        threshold: 20%
 
       - type: patch
         target: auto
         threshold: 10%
 
   individual_components:
 
-    - component_id: scripts  # this is an identifier that should not be changed
-      name: scripts  # this is a display name, and can be changed freely
-      # The list of paths that should be in- and excluded in this component:
-      paths: ["scripts/**", "!scripts/examples/**", "!**/test_*.py"]
-
-    - component_id: scripts/examples
-      name: scripts/examples
-      paths: ["scripts/examples/**", "!scripts/**/test_*.py"]
-
     - component_id: ocaml
       name: ocaml
       paths: ["ocaml/**", "!**/test_*.py"]
 
-    - component_id: ocaml/xapi-storage
-      name: ocaml/xapi-storage
-      paths:
-        - "ocaml/xapi-storage/**"
-        - "ocaml/xapi-storage-script/**"
-        - "!**/test_*.py"
+    # Not covered by the current tests:
```

Comment:
make test does cover these, maybe something could be done so they actually generate coverage when run?

Reply:
coverage expects to run only python modules. there's a way to change how all python processes being run get hooked with coverage, but it's not trivial to set up: https://coverage.readthedocs.io/en/7.4.4/subprocess.html